modules/fw/manifests/roles/dep_signing_from_osx.pp

```diff
@@ -8,10 +8,6 @@
     fw::rules { 'allow_dep_signing_from_osx':
         # All signing sources
         sources =>  [   $::fw::networks::all_partner_repack,
```

Comment:
do we need `all_partner_repack` anymore? in this file and the other `*signing_from_*` files.

Reply:
Nope, I'll remove it.